client/src/Movies/MovieList.js

```diff
@@ -31,7 +33,9 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
+        <Link to ={`/movies/${movie.id}`}>
       <h2>{title}</h2>
+        </Link>
```

Comment:
I think MVP specified linking the whole movie card. but this is okay with me as long as it works!